Changes

```diff
@@ -3,6 +3,9 @@ Working version
 
 ### Language features:
 
+* #12295: Give `while true' a polymorphic type, similarly to 'assert false'
```

Comment:
The initial single quote are different in both quotations :)

Reply:
Thanks! Now fixed.  I've also listed reviewers and reporters.